compiler/optimizer/OMROptimizer.hpp

```diff
@@ -317,6 +331,13 @@ class Optimizer
    bool                          _isIlGen;
 
    const OptimizationStrategy *          _strategy;
+
+   /* 
+    * Since mock strategies are only used in testing right now, we make this
+    * static to ease implementation. 
+    */
```

Comment:
I think a summary of this blurp from the commit as a comment here will help future archaeologists understand why you're making it static:
```
This is currently not a thread-safe implementation beause doing a
thread-safe implementation would require a more invasive compilation
control modification (since the strategy would need to be injected into
the Compilation object, but due to where it's created, there's little
opportunity for this.)
```